### PR TITLE
Added Activity Log package

### PR DESCRIPTION
**Pull request by promatik**

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -19,6 +19,8 @@ public function boot()
             'monster' => 'App\Models\Monster',
             'user'    => 'App\User',
         ]);
+
+        // CauserResolver::setCauser(User::first());
```
Comment (pxpm) [inline app/Providers/AppServiceProvider.php:23]: is this needed ? a leftover or an example ?

Comment (promatik) [reply app/Providers/AppServiceProvider.php]: Leftover...
Activity Log package now handles that by it self 👌

File: app/Http/Controllers/Admin/MonsterCrudController.php [hunk 2]
```diff
@@ -16,6 +16,8 @@ class MonsterCrudController extends CrudController
     use \Backpack\CRUD\app\Http\Controllers\Operations\FetchOperation;
     use \Backpack\Pro\Http\Controllers\Operations\DropzoneOperation { dropzoneUpload as traitDropzone; }
     use \App\Http\Controllers\Admin\Operations\SMSOperation; //Custom Form Operation Example
+    use \Backpack\ActivityLog\Http\Controllers\Operations\ShowModelActivityLogsOperation;
```
Comment (pxpm) [inline app/Http/Controllers/Admin/MonsterCrudController.php:19]: I am just wondering .. if the namespace is already `\Backpack\ActivityLog\` we could probably avoid a name so verbose. I I was thinking in something like `ModelActivityOperation` and `EntryActivityOperation` ?

Comment (promatik) [reply app/Http/Controllers/Admin/MonsterCrudController.php:19]: Do we also always use `...\Http\Controllers\Operations\...`?
Is this part required?

Comment (pxpm) [reply app/Http/Controllers/Admin/MonsterCrudController.php:19]: That's probably better for consistency accross operations in all packages.

Comment (promatik) [reply app/Http/Controllers/Admin/MonsterCrudController.php:19]: Ok, so should we rename;
`ShowModelActivityLogsOperation`, to;
`ModelActivityOperation`

Comment (tabacitu) [reply app/Http/Controllers/Admin/MonsterCrudController.php:19]: > That's probably better for consistency accross operations in all packages.

Agreed!

> Ok, so should we rename to `ModelActivityOperation`

Agreed! Good catch @pxpm and good name suggestion! MUCH better!

**Pull request merged**